dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/MainWindow.xaml.cs

```diff
@@ -34,24 +33,25 @@ namespace VisualLayerIntegration
     /// </summary>
     public partial class MainWindow : Window
     {
-        private Random random = new Random();
-        private string chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
-        private string[] customerFirstNames = new string[] { "Angel", "Josephine", "Wade", "Christie", "Whitney", "Ismael", "Alexandra", "Rhonda", "Dawn", "Roman", "Emanuel", "Evan", "Aaron", "Bill", "Margaret", "Mandy", "Carlton", "Cornelius", "Cora", "Alejandro", "Annette", "Bertha", "John", "Billy", "Randall" };
-        private string[] customerLastNames = new string[] { "Murphy", "Swanson", "Sandoval", "Moore", "Adkins", "Tucker", "Cook", "Fernandez", "Schwartz", "Sharp", "Bryant", "Gross", "Spencer", "Powers", "Hunter", "Moreno", "Baldwin", "Stewart", "Rice", "Watkins", "Hawkins", "Dean", "Howard", "Bailey", "Gill" };
+        private static readonly Random random = new Random();
+        private static readonly string chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+        private static readonly string[] customerFirstNames = new string[] { "Angel", "Josephine",
```

Comment:
>string [](start = 66, length = 6)

the "string" is redundant on array initializers when the type is obvious. Just use "new [] { "Angel", "Josephine", ...